frontend/src/components/translation/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
Instead of this, can `navigation\Header.tsx` be modified to optionally accept a story title? Story title aside, the two headers should be the same